### ISSUE TITLE
Tab completion on just $ breaks PSReadLine

### ISSUE THREAD

**Issue by pavelbraginskiy** [PowerShell/PSReadLine#714]
![image](https://user-images.githubusercontent.com/29113974/41559600-538f80b4-72f9-11e8-9777-fa3108a1c02d.png)

Environment data
----------------
```
PS version: 6.1.0-preview.2
PSReadline version: 2.0.0-beta2
os: Linux ThinkPad 4.16.15-300.fc28.x86_64 #1 SMP Tue Jun 12 00:42:35 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
PS file version: 
```

Steps to reproduce or exception report
--------------------------------------
```
Last 3 Keys:
 $ Tab y

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was -2.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.Menu.DrawMenu(Menu previousMenu)
   at Microsoft.PowerShell.PSConsoleReadLine.PossibleCompletionsImpl(CommandCompletion completions, Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.CompleteImpl(Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------


```

**Comment by vors** [comment]
I can repro it on 2.0.0 beta 1 with Emacs mode.
Looks like some overflow when possible completion options don't fit on one screen.

**Comment by bgshacklett** [comment]
I believe this may be captured in #663.

**Comment by erichiller** [comment]
I think I have this same issue, I'd rather not duplicate ticket.
At $HOMEPROFILE dir I did cat `C:\users\<me>\.ssh\<tab><tab>`

And Received:

```powershell
 UpArrow Delete Delete Delete Delete Delete Delete Delete Delete Delete Delete Delete Delete p i n g Space Enter
 s s h Space v m Backspace Backspace Backspace Backspace Backspace Backspace c a t Space ~ / . s s h Tab Tab Tab Tab Tab
```

Exception:
```
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: top
Actual value was 9999.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.Menu.DrawMenu(Menu previousMenu, Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.PossibleCompletionsImpl(CommandCompletion completions, Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.CompleteImpl(Boolean menuSelect)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

**Comment by lzybkr** [comment]
I *think* this was fixed in [2.0.0-beta3](https://www.powershellgallery.com/packages/PSReadLine/2.0.0-beta3) - maybe you can give it a try?

**Comment by erichiller** [comment]
Again, apologies for the delay @lzybkr - It _does_ work when running vanilla powershell now. If I use `oh-my-posh` then it appears the script block is not being processed or some such, the second time around. I put an issue there with far more detail if you think this is related. [oh-my-posh 116](https://github.com/JanDeDobbeleer/oh-my-posh/issues/116)
